lecture/forms.py

```
@@ -1,11 +1,19 @@
 from django import forms
 from .models import Lecture, LecturePDF
+from django.utils.text import slugify
 
 
 class LectureForm(forms.ModelForm):
     class Meta:
         model = Lecture
-        fields = ['title', 'course']
+        fields = ['course','module','title','description','video_url','content','order']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)